fastestimator/trace/metric/bleu_score.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2022 The FastEstimator Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Counter, Iterable, Union
+
+import numpy as np
+from nltk.translate.bleu_score import Fraction, SmoothingFunction, brevity_penalty, corpus_bleu, math, \
```

Comment:
does nltk actually implement a 'math' package or is this really the python math library?

Reply:
Its just python math library.

Reply:
let's import math directly then instead of getting it via nltk